integration_tests/integration/link/requestLink.spec.ts

```diff
@@ -0,0 +1,86 @@
+import Page from '../../pages/page'
+import RequestLinkPage from '../../pages/link/requestLink'
+import EmailSentPage from '../../pages/link/emailSent'
+
+context('Request Link', () => {
+  beforeEach(() => {
+    cy.task('reset')
+  })
+
+  it('should render request link page to unauthenticated user and reset cookie', () => {
+    cy.setCookie('create_barcode_token', '... a barcode ...')
+    cy.visit('/link/request-link')
+    Page.verifyOnPage(RequestLinkPage)
+
+    cy.getCookie('create_barcode_token').should('not.exist')
+  })
+
+  it('html field validation should prevent form submission given invalid email address', () => {
+    cy.visit('/link/request-link')
+    let page = Page.verifyOnPage(RequestLinkPage)
+
+    page.enterEmailAddress('an-Invalid-Email-Address')
+    page.submitForm()
+
+    page = Page.verifyOnPage(RequestLinkPage)
+    page.emailField().then($input => {
+      expect(($input[0] as HTMLInputElement).validationMessage).to.eq(
+        "Please include an '@' in the email address. 'an-Invalid-Email-Address' is missing an '@'."
+      )
```

Comment:
It would be nice to move this onto the Cypress page

Reply:
Agreed, its a bit ugly where I've done it at the moment. Next PR 👍